sdk/javascript/generator/printers.py

```diff
@@ -217,13 +224,17 @@ def get_default_value(self):
 	def get_size(self):
 		return f'this.{self.name}.size'
 
-	def load(self, buffer_name='view.buffer'):
-		if DisplayType.STRUCT == self.descriptor.display_type and self.descriptor.is_abstract:
+	def load(self, buffer_name='view.buffer', is_aligned=False):
+		display_type = self.descriptor.display_type
+		if DisplayType.STRUCT == display_type and self.descriptor.is_abstract:
 			# HACK: factories use this printers as well, ignore them
 			if 'parent' != self.name:
 				factory_name = self.get_type() + 'Factory'
 				return f'{factory_name}.deserialize({buffer_name})'
 
+		if is_aligned and display_type in [DisplayType.INTEGER, DisplayType.ENUM]:
```

Comment:
tuple ? (i forget if there's a reason we use list instead of tuple in many places 😅 )

Reply:
I usually forget tuple can be used in statements like this